common/key/store.go

```diff
@@ -213,11 +212,11 @@ func SelfSignAll(l log.Logger, multibeaconFolder string) error {
 			return fmt.Errorf("beacon id [%s] - error loading private/public: %w", beaconID, err)
 		}
 
-		// migration path: if a group wasn't reshared in a long time, its secret share won't containt the scheme name
+		// migration path: if a group wasn't reshared in a long time, its secret share won't contain the scheme name
 		// but the group will.
 		group, err := fs.LoadGroup()
 		if err != nil {
-			l.Warnw("could not load group, please report this: %w", "beaconID", beaconID, "err", err)
+			l.Warnw("could not load group, please report this", "beaconID", beaconID, "err", err)
```

Comment:
this log happens on a fresh node, so perhaps we should remove 'please report this'